ffi/examples/read-table/schema.h

```diff
@@ -106,49 +108,53 @@ void visit_struct(
   void* data,
   uintptr_t sibling_list_id,
   struct KernelStringSlice name,
+  bool is_nullable,
   uintptr_t child_list_id)
 {
   SchemaBuilder* builder = data;
   char* name_ptr = allocate_string(name);
   PRINT_CHILD_VISIT("struct", name_ptr, sibling_list_id, "Children", child_list_id);
-  SchemaItem* struct_item = add_to_list(&builder->lists[sibling_list_id], name_ptr, "struct");
+  SchemaItem* struct_item = add_to_list(&builder->lists[sibling_list_id], name_ptr, "struct", is_nullable);
   struct_item->children = child_list_id;
 }
+
 void visit_array(
   void* data,
   uintptr_t sibling_list_id,
   struct KernelStringSlice name,
-  bool contains_null,
+  bool is_nullable,
   uintptr_t child_list_id)
 {
   SchemaBuilder* builder = data;
   char* name_ptr = malloc(sizeof(char) * (name.len + 24));
   snprintf(name_ptr, name.len + 1, "%s", name.ptr);
-  snprintf(name_ptr + name.len, 24, " (contains null: %s)", contains_null ? "true" : "false");
+  snprintf(name_ptr + name.len, 24, " (is nullable: %s)", is_nullable ? "true" : "false");
```

Comment:
I think we only have 22 byte max now? (it's either 21 or 22 bytes depending on if it's true/false)